engine/cleanupmanager/update.go

```diff
@@ -98,7 +98,7 @@ func (db *DB) update(ctx context.Context) error {
 	lookup := lookupMap(currentUsers)
 	for _, dat := range m {
 		cleanupScheduleData(&dat.Data, lookup, now)
-		rawData, err := json.Marshal(dat)
+		rawData, err := json.Marshal(dat.Data)
```

Comment:
No, because `m` is of type `[]schedData` which is a slice of structs (non-pointers), if it was `[]*schedData` then it would be possible that `dat` was `nil` and cause a panic, but there's no possibility of that here.

Reply:
Gotcha, thanks for the clarification :)